Tavisca.Bootcamp.LanguageBasics.Exercise1/Equation.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1{
+class Equation{
+    public Equation(string operand1,string operand2,string operation,string result){
+        Operand1=operand1;
+        Operand2=operand2;
+        Operation=operation;
+        Result=result;
+    }        
+    public string Operand1 { get; private set; }
+    public string Operand2 { get; private set; }
+    public string Operation { get; private set; }
+    public string Result { get; private set; }
+    private bool IsValidValue(string actualValue,string expectedValue){
+        if(string.IsNullOrEmpty(actualValue)||string.IsNullOrEmpty(expectedValue))
+            return false;
+        if(expectedValue.Length==actualValue.Length)
```

Comment:
Make sure do give one space before and after each operator. It makes code more readable